packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -670,6 +672,10 @@ export const style = createTheme({
     borderEndWidth: createRenamedProperty('borderInlineEndWidth', borderWidth),
     borderTopWidth: borderWidth,
     borderBottomWidth: borderWidth,
+    borderInlineStartColor: borderColor, // don't know why i can't use renamed
+    borderInlineEndColor: borderColor,
```

Comment:
If we are keeping these then they should be named like our other logical properties: `borderStartColor`, `borderEndColor`

Reply:
yeah, as the comment says, i don't know why I can't use the createRenamedProperty like I could for borderEndWidth

Reply:
![Screenshot 2025-01-29 at 1 51 52 pm](https://github.com/user-attachments/assets/7c176e39-ced9-4cca-9f73-7a11ce8e6f1b)